### PR TITLE
Magiciens des mathématiques: composants

### PR DESCRIPTION
**Pull request by fredo509**
In this millestone I :

-Add a React component  
- Design Calculator.js

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+export default function theCalculator() {
```
Comment (Shisui6) [inline src/components/calculator.js:3]: Please delete `logo192.png` and `logo512.png` from the public directory. These are boilerplate files🙏

image

File: src/components/calculator.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  return (
```
Comment (Shisui6) [inline src/components/calculator.js:4]: Please delete `logo.svg` from the src directory. These are boilerplate files🙏

image

File: src/components/calculator.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        <p>Math Magicians :) </p>
+      </header>
+      <main>
+        <section>
+          <div id="Myscreen">
+            <input id="screen" value="0" readOnly />
+          </div>
+          <div id="keyboard">
+            <div className="btnBox">
+              <button id="btnAC" type="button"> AC </button>
```
Comment (Shisui6) [inline src/components/calculator.js:18]: Kindly update the colors of your calculator component to match the given design. The colors used are `#f5913e`, `#e0e0e0`, and `#858694`

Your design:
image

Given design:
image

File: src/components/calculator.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+            </div>
+            <div className="btnBox">
+              <button id="btn+/-" type="button"> +/- </button>
```
Comment (Shisui6) [inline src/components/calculator.js:21]: According to the project requirements, the calculator component should have at least 1 child component. Please update your calculator component to include a child component🙏. You can try to make the buttons a component that you can import or add any other component

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @fredo509, 

# Requires Changes 🍀
You have done a great job so far!👏👏
Your design is good 🖼️, and your code follows best practices. However, there are a few issues that you still need to work on🪲.

## Highlights
- Calculator component created ✅
- App works ✅


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calculator.js (hunk 2 above)
Comment (danmainah) [reply src/components/calculator.js:4]: - [ ] @fredo509 great work implementing the changes requested however you still have not deleted logo.svg from the src directory as previously requested

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @fredo509 , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @fredo509 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights

✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**